chess/ChessGameModel.swift

```diff
@@ -176,6 +187,14 @@ extension ChessGameModel {
 
 extension ChessGameModel {
 
+    enum MessageType {
```

Comment:
Type is a reserved word in swift so I would recommend to avoid using it
```suggestion
    enum MessageKind {
```